### DISCUSSION TITLE
Add NPC's to the Hourglass

### DISCUSSION THREAD

**Discussion by Vasculio294** [exult/exult#367] [General]
I have no scripting knowledge. But i am curious... if there was a way to add NPC's to the Hourglass of Fate. I would consider Boydon and Wilfred as examples.

**Comment by KnightCaptainU7** [comment]
It should be possible. See https://github.com/exult/exult/blob/master/content/sifixes/src/misc/resurrect.uc for how the list is constructed.

**Comment by Vasculio294** [comment]
I think I might have found my answer to the scripting. Thanks for the resurrect uc.
